### ISSUE TITLE
Invalid signature when using Oauth 1 and method POST with x-www-form-urlencoded

### ISSUE THREAD

**Issue by hartono-sulaiman-kaskus** [a85/POSTMan-Chrome-Extension#1838]
Hi

I got invalid signature when use Oauth 1, POST with content-type `application/x-www-form-urlencoded`

My investigation found out that Postman does not include form field in base string when creating signature 

http://oauth.net/core/1.0a/#anchor13

```
Parameters in the HTTP POST request body (with a content-type of application/x-www-form-urlencoded).
```

i use postman 

```
4.0.2
( OS X / x86-64 )
( Chrome 48.0.2564.116 )
```

**Comment by abhijitkane** [comment]
@hartono-sulaiman-kaskus Thanks for reporting this. We'll push a fix for this soon.

**Comment by abhijitkane** [comment]
@hartono-sulaiman-kaskus 
Can you check if this has been fixed in the latest version of the app - 4.1.2?
